packages/editor/src/editor-styles.scss

```diff
@@ -144,3 +144,17 @@ ol ul {
 		max-width: 1100px;
 	}
 }
+
+a {
+	color: inherit;
+	transition: none;
+}
+
+code,
+kbd {
+	padding: 0;
+	margin: 0;
+	background: inherit;
+	font-size: inherit;
+	font-family: monospace;
+}
```

Comment:
What about the themes that provide editor styles but not for these tags? We'll have the admin styles applied, shouldn't we revert them for these themes?

Reply:
For those themes, this PR should actually be even better than master because now they receive a better default than just inheriting from wp admin.

Reply:
I don't think this file is loaded for those themes.

Reply:
Oh.

Reply:
Huh, this file is not loaded? So then there is admin CSS bleed?

Reply:
yes, this file is the "default editor styles" so it's only loaded when the theme doesn't define any editor styles IIRC.

Reply:
Hm, no these are admin normalisation styles, right? These are supposed to always load.

Reply:
I currently have a theme with editor styles, and this stylesheet loads as well. Am I missing something?

Reply:
Maybe it changed then but I'm pretty sure the original purpose was to act as default styles. If these are loaded consistently, I don't see any blockers for this PR.

Reply:
I confirmed that these styles are loaded for themes with editor styles.